php-fpm/recipes/cloudwatch.rb

```diff
@@ -0,0 +1,23 @@
+
+cronscript = "#{node['php-fpm']['prefix']}/bin/phpfpm-cloudwatch.sh"
+
+include_recipe 'awscli'
+
+template cronscript do
+  source 'cloudwatch.sh.erb'
+  mode   '0755'
+  owner  'root'
+  group  'root'
+  variables(
+    'instancename' => get_record_name.gsub(/\.|\W/, '_'),
+    'stackname' => get_normalized_cluster_name.gsub(/\.|\W/, '_'),
+    'region' => node['opsworks']['instance']['region']
+  )
+end
+
+cron_d 'phpfpm-cloudwatch' do
```

Comment:
@gilleyj it would be mighty fine if you adjusted the specs and added one for the default (`false`) also, let me know if I can help there or give pointers!